dashboard/src/actions/overviewActions.js

```diff
@@ -28,7 +30,10 @@ export const getDatasets = () => async (dispatch, getState) => {
     params.append("metadata", "global");
     params.append("metadata", "user");
 
-    params.append("mine", "true");
+    if (loggedIn) {
+      params.append("mine", "true");
+    }
```

Comment:
The "Overview" page was always supposed to be focused on "my" data, so I'm not so sure about this change. Although it's also the only view that currently support displaying *metadata*, and I'd really like to open that for all data.

On the other hand, many of the controls (like saved/seen/publish) that appear in the overview page are meaningless for public datasets that aren't yours: you don't have rights to set those metadata values or to update the dataset visibility.